python/pyarrow/tests/test_flight.py

```diff
@@ -355,17 +355,20 @@ def slow_stream():
 class ErrorFlightServer(FlightServerBase):
     """A Flight server that uses all the Flight-specific errors."""
 
+    errors = {
+        "internal": flight.FlightInternalError,
```

Comment:
@lidavidm `flight` is `None` when we don't enable Flight.

See also: https://github.com/ursacomputing/crossbow/runs/6046421802?check_suite_focus=true#step:6:2372

```text
==================================== ERRORS ====================================
_ ERROR collecting usr/local/lib/python3.8/dist-packages/pyarrow/tests/test_flight.py _
usr/local/lib/python3.8/dist-packages/pyarrow/tests/test_flight.py:355: in <module>
    class ErrorFlightServer(FlightServerBase):
usr/local/lib/python3.8/dist-packages/pyarrow/tests/test_flight.py:359: in ErrorFlightServer
    "internal": flight.FlightInternalError,
E   AttributeError: 'NoneType' object has no attribute 'FlightInternalError'
```

Reply:
Sorry about that. See #12911/ARROW-16216.